MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+"""Bazel build and test dependencies."""
+
+# NOTE: When editing this file, also update the lockfile.
+#   bazel mod deps --lockfile_mode=update
+
+module(
+    name = "rules_platform",
+    repo_name = "rules_platform",
```

Comment:
the repo_name line is redundant -- it defaults to the same as `name`

Reply:
Removed. Thanks!